Tools/build/generate_sbom.py

```diff
@@ -60,7 +60,7 @@ class PackageFiles(typing.NamedTuple):
         exclude=[
             "Modules/expat/expat_config.h",
             "Modules/expat/pyexpatns.h",
-            "Modules/_hacl/refresh.sh",
+            "Modules/expat/refresh.sh",
```

Comment:
@SethMichaelLarson If I understand correctly, this was a typo.
Could you confirm?
Can we simply exclude the file now, like this?

Reply:
Considering that we have an entry:

```py
    "hacl-star": PackageFiles(
        include=["Modules/_hacl/**"],
        exclude=[
            "Modules/_hacl/refresh.sh",
            "Modules/_hacl/README.md",
            "Modules/_hacl/python_hacl_namespace.h",
        ]
),
```

I think we can say that it was a typo.

Reply:
Yes this looks like a typo! FYI my GitHub handle is `@sethmlarson`